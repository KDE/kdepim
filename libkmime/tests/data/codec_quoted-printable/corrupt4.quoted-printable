Two =3D at the very end of the encoded data: ==