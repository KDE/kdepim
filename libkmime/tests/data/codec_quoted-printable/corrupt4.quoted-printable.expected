Two = at the very end of the encoded data: ==